aria/modeling/types.py

```diff
@@ -23,7 +23,7 @@
 )
 from sqlalchemy.ext import mutable
 
-from .. import exceptions
+from ..storage import exceptions
```

Comment:
I think we should create new exceptions - modelling exceptions. from within the same package it think it's better if we trow package specific exceptions.